### PR TITLE
:white_check_mark: feat:add media range syntax format

### PR DESCRIPTION
**Pull request by hoelshen**
Close:  #3984

-   Chrome 104 started to support: https://chromium-review.googlesource.com/c/chromium/src/+/3634678
-   ls: https://www.w3.org/TR/mediaqueries-4/#typedef-media-query-list

### CODE REVIEW

File: features-json/css-media-range-syntax.json [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+      "75":"y",
+      "76":"y",
+      "77":"y",
+      "78":"y",
+      "79":"y",
+      "80":"y",
+      "81":"y",
+      "82":"y",
+      "83":"y",
+      "84":"n",
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:155]: `y`?

File: features-json/css-media-range-syntax.json [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+      "10":"n",
+      "11":"n",
+      "11.1":"n",
+      "11.5":"n",
+      "12":"n",
+      "12.1":"n",
+      "64":"n"
+    },
+    "and_chr":{
+      "103":"y"
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:454]: `n`

File: features-json/css-media-range-syntax.json [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+    },
+    "and_ff":{
+      "101":"n"
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:457]: `y`

**Comment by Schweinepriester** [comment]
Related: https://github.com/Fyrd/caniuse/issues/3984

File: features-json/css-media-range-syntax.json (hunk 2 above)
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:454]: ok

File: features-json/css-media-range-syntax.json (hunk 3 above)
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:457]: ok

File: features-json/css-media-range-syntax.json (hunk 1 above)
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:155]: yes，y

File: features-json/css-media-range-syntax.json [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+  },
+  "notes":"",
+  "notes_by_num":{
+ 
+  },
+  "usage_perc_y":68.85,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"",
+  "keywords":"",
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:500]: Please add the keywords, which are used when searching at caniuse.com.

`@media,media queries,range,range syntax,range context`

File: features-json/css-media-range-syntax.json [hunk 5]
```diff
@@ -0,0 +1,506 @@
+{
+  "title":"media-query-range-syntax",
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:2]: Media Queries: Range Syntax

File: features-json/css-media-range-syntax.json [hunk 6]
```diff
@@ -0,0 +1,506 @@
...
+  "description":"The Media Queries Level 4 specification includes some syntax improvements to make media queries using features that have a 'range' type, for example width or height, less verbose. Level 4 adds a range context for writing such queries.",
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:3]: > Level 4 adds a range context for writing such queries

Change to:

Can be used ordinary mathematical comparison operators: `>`, '<', `>=`, or '<='. For example:

Without this feature: `@media (min-width: 100px) and (max-width: 1900px)`
With this feature: `@media (100px <= width <= 1900px)`

Comment (yisibl) [reply features-json/css-media-range-syntax.json:3]: It is best if we can show the user how this syntax is used at a glance in the description.

File: features-json/css-media-range-syntax.json [hunk 7]
```diff
@@ -0,0 +1,506 @@
+{
+  "title":"media-query-range-syntax",
+  "description":"The Media Queries Level 4 specification includes some syntax improvements to make media queries using features that have a 'range' type, for example width or height, less verbose. Level 4 adds a range context for writing such queries. For example, using the max- functionality for width we might write the following",
+  "spec":"",
+  "status":"other",
```
[the remark this answers is not in the document]
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:5]: done

[the remark this answers is not in the document]
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:5]: done

File: features-json/css-media-range-syntax.json [hunk 8]
```diff
@@ -0,0 +1,506 @@
...
+      "url": "https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries",
+      "title": "Syntax improvements in Level 4" 
+    },
+    {
+      "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1422225#c55",
+      "title": "Firefox support bug"
+    },
+    {
+      "url": "https://www.w3.org/TR/mediaqueries-4/#typedef-media-query-list",
+      "title": "Media Query List"
```
[the remark this answers is not in the document]
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:17]: done

File: features-json/css-media-range-syntax.json [hunk 9]
```diff
@@ -0,0 +1,506 @@
...
+    }
```
[the remark this answers is not in the document]
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:18]: done

File: features-json/css-media-range-syntax.json (hunk 4 above)
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:500]: done

File: features-json/css-media-range-syntax.json (hunk 5 above)
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:2]: done

File: features-json/css-media-range-syntax.json [hunk 10]
```diff
@@ -0,0 +1,506 @@
...
+  "notes_by_num":{
+ 
+  },
+  "usage_perc_y":68.85,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"",
+  "keywords":"",
+  "ie_id":"",
+  "chrome_id":"",
```
[the remark this answers is not in the document]
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:502]: done

File: features-json/css-media-range-syntax.json (hunk 6 above)
Comment (hoelshen) [reply features-json/css-media-range-syntax.json:3]: Strongly Agree

**Comment by yisibl** [comment]
LGTM +1

File: features-json/css-media-range-syntax.json [hunk 11]
```diff
@@ -0,0 +1,518 @@
...
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+ 
+  },
+  "usage_perc_y":68.85,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"",
```
Comment (yisibl) [inline features-json/css-media-range-syntax.json:511]: "parent":"css-mediaqueries"

Comment (hoelshen) [reply features-json/css-media-range-syntax.json:511]: done

**Comment by yisibl** [comment]
@Fyrd Do you have time to look at it? Thanks!

**Comment by Fyrd** [comment]
Thanks! Not strictly necessary since it's already at https://caniuse.com/mdn-css_at-rules_media_range_syntax but good to have the extra information.

**Pull request merged**